### ISSUE TITLE
Unable to retrieve XSRF-TOKEN cookie from Laravel Sanctum API using Chrome extension

### ISSUE THREAD

**Issue by creativeLogics** [laravel/sanctum#435]
### Sanctum Version

v3.0.1

### Laravel Version

v9.45.1

### PHP Version

v8.1.13

### Description

Hi,

### Description
I'm trying to retrieve the XSRF-TOKEN cookie from my Laravel Project which is using Laravel Fortify with Laravel Sanctum API using a fetch call from a Chrome extension, but I'm unable to do so. I've followed the instructions in the Laravel Sanctum documentation and have set the SANCTUM_STATEFUL_DOMAINS variable in my .env file correctly, but I'm still encountering issues.

I've verified that my server is properly configured to allow cross-origin requests by checking the response headers of my API requests in my browser's developer tools. The Access-Control-Allow-Origin header is set to the origin of my Chrome extension.

However, it seems that my server's CORS settings may not be allowing the X-XSRF-TOKEN header to be sent in requests. I've tried adding the X-XSRF-TOKEN header to my server's CORS configuration using the Access-Control-Allow-Headers header, but this hasn't resolved the issue. 

You can check the screenshot of the network tab request "api/sanctum/csrf-cookie" xcsrf-token is in the response header but there is also a warning icon.
![Screenshot_2](https://user-images.githubusercontent.com/49434107/230880665-8772fa5a-71f5-4633-8cbe-9db07cb551f0.png)

It might be, SESSION_DOMAIN issue. since it's a Chrome extension it doesn't have any domain.

Any assistance in resolving this issue would be greatly appreciated.

**Steps to reproduce**
Make a fetch call to the sanctum/csrf-cookie endpoint with the credentials option set to 'include' from Chrome extension,

**Additional information**
Laravel version: 8.60.0
PHP version: 8.0.7
Operating system: Windows 10

### Steps To Reproduce

**Steps to reproduce**
Make a fetch call to the sanctum/csrf-cookie endpoint with the credentials option set to 'include' from the Chrome extension,

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by creativeLogics** [comment]
Hi there,

Thanks for your response. I understand that this may be more of a question than a bug report. However, I have followed the instructions provided in the Laravel Sanctum documentation and have configured my server and Chrome extension as instructed, but I'm still encountering issues.

I've posted this issue on the Laravel Sanctum GitHub repository in hopes of receiving further assistance from the Laravel community. 
Thank you for your understanding and I appreciate any help or guidance anyone can provide.

Best regards.
